CHANGELOG.md

```diff
@@ -12,6 +12,22 @@ The minor version will be incremented upon a breaking change and the patch versi
 
 ### Features
 
+### Fixes
+
+### Breaking
+
+## [0.32.0] - 2025-10-01
+
+### Features
+
+### Fixes
+
+### Breaking
+
+## [0.32.0] - 2025-10-01
```

Comment:
Looks like the [`bump-version`](https://github.com/solana-foundation/anchor/blob/d5d7eb97979234eb1e9e32fcef66ce171a928b62/bump-version.sh) script has been run twice, which ended up adding double entry here.